namespaces/live.cloud-platform.service.justice.gov.uk/care-arrangement-plan-prototype/resources/ecr.tf

```diff
@@ -0,0 +1,17 @@
+module "ecr-repo" {
```

Comment:
Possibly this should use `_` @josephsilcock

Reply:
In fact, if you rename to ecr_credentials it will be in keeping with some of our other namespaces.